server/integration-tests/src/test/java/io/smallrye/graphql/tests/client/dynamic/DynamicClientInjectionTest.java

```diff
@@ -50,7 +50,7 @@ public static WebArchive deployment() {
     }
 
     @Inject
```

Comment:
At some point (not now) we can maybe allow this without `@Inject` ? w.d.y.t ?

Reply:
@phillip-kruger Quarkus allows this already, thanks to this build item https://github.com/quarkusio/quarkus/blob/2.0.0.CR2/extensions/smallrye-graphql-client/deployment/src/main/java/io/quarkus/smallrye/graphql/client/deployment/SmallRyeGraphQLClientProcessor.java#L62

Not sure if it's possible for example in WildFly, I guess not

Reply:
OK great. Yea I was talking about Quarkus :)